packages/office-ui-fabric-react/src/components/SpinButton/SpinButton.tsx

```diff
@@ -37,6 +37,9 @@ export type DefaultProps = Required<
   Pick<ISpinButtonProps, 'step' | 'min' | 'max' | 'disabled' | 'labelPosition' | 'label' | 'incrementButtonIcon' | 'decrementButtonIcon'>
 >;
 
+/** Internal only props */
+export type ISpinButtonInternalProps = ISpinButtonProps & DefaultProps;
```

Comment:
does this error if you don't export? the type is only used within the function for casts